src/FakerDotNet/FakerContainer.cs

```diff
@@ -128,7 +130,7 @@ public FakerContainer()
 
             Regexify = new Regexify(this);
         }
-
+       
```

Comment:
This empty whitespace can be trimmed.

You should see if there is a setting in your IDE/editor to trim whitespace when saving the file 😉